frontend/src/DocumentUploadForm.js

```diff
@@ -51,7 +67,7 @@ class DocumentUploadForm extends React.Component {
               <br/>
               Start by uploading a PDF document below!
             </p>
-            <h3>Upload a PDF</h3>
+            <h3>Upload a PDF (Max file size: 80 MB)</h3>
```

Comment:
Could we have it like this

![Screenshot from 2020-03-09 10-19-59](https://user-images.githubusercontent.com/15786604/76221553-a4e88000-61ef-11ea-8e9c-e8332d7d3416.png)